patches/electron+33.0.2.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/node_modules/electron/electron.d.ts b/node_modules/electron/electron.d.ts
```

Comment:
This patch is needed due to an issue in Electron's typing for this module that caused a bug. A fix is waiting to be merged, but for now we'll use this patch.
https://github.com/electron/electron/pull/44391